htdocs/wp-content/themes/defryskemarren/assets/scss/components/_gravityforms.scss

```diff
@@ -0,0 +1,19 @@
+.gform_wrapper {
+	.gfield {
+		.ginput_container_textarea {
+			textarea {
+				&.textarea.small {
+					height: 96px;
+				}
+
+				&.textarea.medium {
+					height: 192px;
+				}
+
+				&.textarea.large {
+					height: 288px;
+				}
+			}
+		}
+	}
+}
```

Comment:
Dit mag wat mij betreft direct in het owc-formulieren parent theme. 

Dit is de legacy gravity forms styling, toch? Dan kan het in `_gravityforms-legacy.scss` 

Daarna approve op deze PR :)